style.css

```diff
@@ -0,0 +1,113 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@media (min-width: 375px) {
+  head {
+    margin-top: 10px;
+  }
+
+  header {
+    padding: 1rem 1.5rem 0 1.5rem;
+  }
+
+  .bgimage {
+    padding: 0 1.5rem 1.5rem 1.5rem;
+    background-image: url("header-shapes mobile.png");
+    background-size: 100%;
+    background-position: cover;
+    background-repeat: no-repeat;
+    height: 100vh;
+    overflow: hidden;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+  }
+
+  .menu a {
+    width: 20.27vw;
+    height: 5.33vw;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 4.8vw;
+    line-height: 74%;
+    color: #6070ff;
+    cursor: pointer;
+    text-decoration: none;
+  }
+
+  .menu {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .menu img {
+    width: auto;
+    height: auto;
+  }
+
+  .intro h1 {
+    font-family: 'Poppins', sans-serif;
+    font-size: 2.5rem;
+    line-height: 2.5rem;
+    margin-bottom: 10px;
+    font-weight: 700;
+    color: #172b4d;
+  }
+
+  .intro p {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5rem;
+    vertical-align: top;
+    color: #344563;
+  }
+
+  .connect h4 {
+    font-family: 'Poppins', sans-serif;
+    color: #7f8cff;
+    font-weight: 500;
+    font-size: 4.25vw;
+    margin: 0.9375rem 0;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 1rem;
+    height: 100vh;
```

Comment:
- [ ] Please adhere to the `Figma` design as closely as possible, even your previous two code reviewer told you through a review that your design is not matching with the `Figma` design, but it seems like you either ignored their comments or may have an issue regarding not understanding their point. But now don't worry I am here to tell you by just removing one line that is currently I highlighted, by removing this line, your design will looks more professional and exactly match with original design. Kindly look at the below table of comparison of requirement and then table of comparison after changes with a video at the end.

### Design Comparison

<table>
<tr>
<th align=“middle”>Your Design</th>
<th align=“middle”>Figma Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-01 20-11-16](https://github.com/Porkalaikarthika/Portfolio/assets/66476329/a654dba8-90dc-47ad-a434-f1e596490e50)

</td>
<td>

![image](https://github.com/Porkalaikarthika/Portfolio/assets/66476329/04d435ab-4e28-4e47-acc9-5ed3c827771b)


</td>
</tr>
</table>

### Design After the change

<table>
<tr>
<th align=“middle”>Before Design</th>
<th align=“middle”>After Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-01 20-11-16](https://github.com/Porkalaikarthika/Portfolio/assets/66476329/a654dba8-90dc-47ad-a434-f1e596490e50)

</td>
<td>

![image](https://github.com/Porkalaikarthika/Portfolio/assets/66476329/17de12fe-22a1-44b1-b113-2b03df30a4ff)


</td>
</tr>
</table>


### Video Explanation

[Screencast from 01-06-23 08:23:06 PM IST.webm](https://github.com/Porkalaikarthika/Portfolio/assets/66476329/de425449-a1de-4fde-96c1-f67e4fd9af9a)